### PR TITLE
B: TechDebt: disable 'morgan' logging for Datadog cost savings

### PR DESCRIPTION
**Pull request by cantremember**
- removed 'logger' from `defaultMiddleware.preludes`
- export `morganMiddleware`, if the caller wants it
- some morgan Telemetry content changes

### CODE REVIEW

File: src/middleware/defaults.spec.ts [hunk 1]
```diff
@@ -1,28 +1,8 @@
-import { Socket } from 'net';
-import _, { noop, omit } from 'lodash';
```
Comment (cantremember) [inline src/middleware/defaults.spec.ts:2]: side-effects of the file-splitting

File: src/middleware/defaults.ts [hunk 2]
```diff
@@ -30,77 +20,10 @@ function _tupleByMiddlewareName(handler: RequestHandlerVariant): [ string, Reque
...
-  immediate: false,
-  stream: process.stdout, // vs. @withjoy/telemetry, because Stream
-});
-
-
 export function getDefaultMiddleware(): DefaultMiddlewareResult {
   const preludesMap = new Map<string, RequestHandlerVariant>([
     corsMiddleware,
     sessionMiddleware(),
-    MORGAN_LOGGER,
```
Comment (cantremember) [inline src/middleware/defaults.ts:103]: removed 'logger' from `defaultMiddleware.preludes`

File: src/middleware/morgan.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    : getProperty(req, SESSION_REQUEST_PROPERTY)
+  );
+  const remoteAddress = (context
+    ? context.remoteAddress
+    : deriveRemoteAddress(req)
+  );
+
+  const loggedData = {
+    source: 'express',
+    action: 'morgan',
```
Comment (cantremember) [inline src/middleware/morgan.ts:56]: some morgan Telemetry content changes:  the action

*️⃣ ... which will have an impact on some Datadog Dashboards

File: src/middleware/morgan.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    sessionId,
+
+    // outbound (because { immediate: false })
+    statusCode: tokens.status(req, res),
+    contentLength: tokens.res(req, res, 'content-length'),
+    responseTimeMs: tokens['response-time'](req, res),
+  };
+
+  // with Telemetry, at 'info' level
+  const message = `morgan: '${ loggedData.method } ${ loggedData.path || '/' }' => ${ loggedData.statusCode || 0 }`;
```
Comment (cantremember) [inline src/middleware/morgan.ts:72]: morgan Telemetry content change:  a nice message (vs. "morgan" all the damned time)

**Review by cantremember — COMMENTED** [review]
## annotated

**Review by ForestBeaver — APPROVED** [review]
✅ looks good, logs are about to get much less noisy.

**Pull request merged**